fetch-vault-certificate/action.yaml

```diff
@@ -0,0 +1,47 @@
+name: generate a certificate with vault
+description: Generates a certificate with vault
+
+inputs:
+  vault_token:
+    description: The vault token
+    required: true
+  vault_addr:
+    description: The vault addr
+    required: true
+  pki_engine_path:
+    description: The path in vault to the PKI engine
+    required: true
+  common_name:
+    description: The CA common name
+    required: false
+    default: cloud.armory.io
+  key_filename:
+    description: The file name (including path) to write the certificate private key (.key) to
+    required: true
+  certificate_filename:
+    description: The file name (including path) to write the certificate (.crt) to
+    required: true
+  ttl:
+    description: The ttl for the generated certificate
+    required: false
+    default: 1h
+
+runs:
+  using: composite
+  steps:
+    - name: Generate cert from vault
+      env:
+        VAULT_ADDR: ${{ inputs.vault_addr }}
+        VAULT_TOKEN: ${{ inputs.vault_token }}
+        AWS_REGION: us-west-2
```

Comment:
shouldn't region also be parametrized?

Reply:
It could be, but vault is running there, so it has to be us-west-2